src/AWS.Deploy.Common/Recipes/RecipeDefinition.cs

```diff
@@ -89,6 +89,11 @@ public class RecipeDefinition
         /// </summary>
         public int RecipePriority { get; set; }
 
+        /// <summary>
+        /// Flag to indicate if this recipe is generated from a custom recipe snapshot.
+        /// </summary>
+        public bool IsCustomRecipe { get; set; }
```

Comment:
When creating a deployment project we should not allow users to select already saved deployment projects. Can you confirm if that is being done and if so is it by this project? I think we also want a more meaningful name like `UsingSavedDeploymentProject`. We eventually want users to distribute their recipes which would be a different form of "custom" recipe.

Reply:
verified the the user cannot select an already saved deployment project when creating a deployment project. Also changed `IsCustomRecipe` to `CreatedFromSavedDeploymentProject`